aws/sdk/examples/snowball/src/bin/create-address.rs

```diff
@@ -70,34 +93,73 @@ async fn main() -> Result<(), aws_sdk_snowball::Error> {
         street1,
         street2,
         street3,
+        verbose,
     } = Opt::from_args();
 
     let region_provider = region::ChainProvider::first_try(region.map(Region::new))
         .or_default_provider()
         .or_else(Region::new("us-west-2"));
 
+    println!();
+
+    if verbose {
+        println!("Snowball version:       {}", PKG_VERSION);
+        println!(
+            "Region:                 {}",
+            region_provider.region().unwrap().as_ref()
+        );
+        println!("City:                   {}", city.clone());
+        println!(
+            "Company:                {}",
+            company.clone().unwrap_or_default()
+        );
+        println!("Country:                {}", country.clone());
+        println!(
+            "Landmark:               {}",
+            landmark.clone().unwrap_or_default()
+        );
+        println!("Name:                   {}", name.clone());
+        println!("Phone number:           {}", phone_number.clone());
+        println!("Postal code:            {}", postal_code.clone());
+        println!(
+            "Prefecture or district: {}",
+            prefecture_or_district.clone().unwrap_or_default()
+        );
+        println!("State:                  {}", state.clone());
+        println!("Street1:                {}", street1.clone());
+        println!(
+            "Street2:                {}",
```

Comment:
can use `{:?}` do use the debug formatter so you don't need to unwrap the options

Reply:
Done.